src/Factories/FunFunFactory.php

```diff
@@ -1283,4 +1292,18 @@ public function newPageNotFoundHTMLPresenter(): PageNotFoundPresenter {
 		return new PageNotFoundPresenter( $this->getLayoutTemplate( '404message.html.twig' ) );
 	}
 
+	public function setServerSideTracker( ServerSideTracker $tracker ) {
+		$this->pimple['server_side_tracker'] = function () use ( $tracker )  {
+			return $tracker;
+		};
+	}
```

Comment:
Question is here: http://stackoverflow.com/questions/40590533/most-semantic-way-to-replace-services-in-pimple-container